src/main/java/com/seleniumeasy/listbox/DataListFilterPage.java

```diff
@@ -0,0 +1,35 @@
+package com.seleniumeasy.listbox;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import com.seleniumeasy.DemoHomePage;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class DataListFilterPage extends DemoHomePage {
+    private final SelenideElement inputSearchAttendees = $("#input-search");
+    private final ElementsCollection companies = $$(".info-block h5"),
+            names = $$(".info-block h4"),
+            titles = $$(".info-block p"),
+            phones = $$x("//span[contains(text(), 'Phone')]"),
+            emails = $$x("//span[contains(text(), 'Email')]");
+
+    public DataListFilterPage enterSearchText(String searchText) {
+        inputSearchAttendees.sendKeys(searchText);
```

Comment:
please do not use Selenium methods
sendKeys is not method of Selenide

Reply:
Replace sendKeys() by setValue()